product/dialogs/service_dialogs/transform-vm.yml

```diff
@@ -0,0 +1,221 @@
+---
+- description: Transform VM
+  buttons: submit,cancel
+  label: Transform VM
+  blueprint_id:
+  dialog_tabs:
+  - description:
+    display: edit
+    label: New tab
+    display_method:
+    display_method_options:
+    position: 0
+    dialog_groups:
+    - description: Import given virtual machine to another infrastructure provider
+      display: edit
+      label: Transform Virtual Machine
+      display_method:
+      display_method_options:
+      position: 0
+      dialog_fields:
+      - name: name
+        description: Name of the created virtual machine
+        data_type:
+        notes:
+        notes_display:
+        display: edit
+        display_method:
+        display_method_options: {}
+        required: true
+        required_method:
+        required_method_options: {}
+        default_value: ''
+        values:
+        values_method:
+        values_method_options: {}
+        options:
+          :protected: false
+        label: Name
+        position: 0
+        validator_type:
+        validator_rule:
+        reconfigurable:
+        dynamic: false
+        show_refresh_button:
+        load_values_on_init:
+        read_only:
+        auto_refresh:
+        trigger_auto_refresh:
+        visible: true
+        type: DialogFieldTextBox
+        resource_action:
+          action:
+          resource_type: DialogField
+          ae_namespace:
+          ae_class:
+          ae_instance:
+          ae_message:
+          ae_attributes: {}
+      - name: provider
+        description: Target infrastructure provider to which this virtual machine
+          should be imported.
```

Comment:
I'm not a native speaker of course, but IMHO the 2 possibilities here are "... provider which this VM should be imported *to*" or "provider *to* which this VM should be imported", but not with 2 "to"s :-)